calicoctl/calico_ctl/node.py

```diff
@@ -68,8 +68,8 @@
   --ip6=<IP6>               The local IPv6 management address to use.
   --ipv4                    Show IPv4 information only.
   --ipv6                    Show IPv6 information only.
-  --libnetwork              Use the libnetwork plugin.
-  --libnetwork-image=<LIBNETWORK_IMAGE_NAME>    Docker image to use for
+  --libnetwork              (Deprecated) Use the libnetwork plugin.
+  --libnetwork-image=<LIBNETWORK_IMAGE_NAME>    (Deprecated) Docker image to use for
```

Comment:
I think replacing "Docker image to use for.." with "This flag will be ignored" is the right thing to do